### ISSUE TITLE
Not working for new update 3095

### ISSUE THREAD

**Issue by rayvenz3** [Eddlm/autosport-racing-system-gta#8]
After update 3095 ARS has stopped working, is it possible for someone to update it to work for the new update?

**Comment by Pizzahut20** [comment]
Yeah same here, it just spawns a driver outside a car, and then nothing happens.

**Comment by kagikn** [comment]
In b3095, `ARS.ARS.SetBrakes()` fails to find the address to extract with the memory pattern `74 0A F3 0F 11 B3 ? ? ? ? EB 25` and tries to dereference the null pointer, and that dereferencing crashes the script for an `NullReferenceException`.

Maybe there's more things to fix, but at least `SetBrakes()` needs another memory pattern to fix the compatibility issues.

**Comment by shifuguru** [comment]
I've just pulled the project 
As long as the author doesn't mind.. I will attempt to update the script to SHVDN3

**Comment by jalalcool12345** [comment]
> I've just pulled the project As long as the author doesn't mind.. I will attempt to update the script to SHVDN3

please fix it this mod is really cool its sad to see it doesnt work anymore

**Comment by Eddlm** [comment]
Eons later..! I updated the pattern. Its not present in 0.8.6 but will be in the next release. Current version works with shvdn nightly, including the braking.
